index.bs

```diff
@@ -285,6 +287,18 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 Perform Logger("warn", <var>data</var>).
 
+<h4 id="dir" oldids="dir-data,dom-console-dir" method for="console">dir(<var>data</var>)</h4>
+
+Let <var>object</var> be an potentially-interactive representation of <var>data</var> treated as a JavaScript object. Perform Printer("log", <var>«object»</var>).
+
+<h4 id="dirxml" oldids="dirxml-data,dom-console-dirxml" method for="console">dirxml(...<var>data</var>)</h4>
+
+<emu-alg>
```

Comment:
That is reasonable. Must've slipped my mind. I think in the end I lean toward processing format specifiers just as `log` does, because `dirxml` behaves so much like `log`